cypress/elements/catalog/products/product-details.js

```diff
@@ -25,7 +25,7 @@ export const PRODUCT_DETAILS = {
   uploadSavedImagesButton: '[data-test-id="upload-images"]',
   uploadMediaUrlButton: '[data-test-id="upload-media-url"]',
   saveUploadUrlButton: '[data-test-id="upload-url-button"]',
-  editVariant: '[id="menu-list-grow"]',
+  editVariant: '[id="row-action-button"]',
```

Comment:
pls change to `editVariant: '[data-test-id="row-action-button"]',`

Reply:
Sorry to fast to furious xD